### PR TITLE
added an "ignore" option for checksumPolicy in documents

### PR DESCRIPTION
**Pull request by sungpeo**
I didn't create a JIRA issue, this is a trivial change.

Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [X] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MNG) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [X] Each commit in the pull request should have a meaningful subject line and body.
 - [X] Format the pull request title like `[MNG-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MNG-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [X] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [X] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [X] You have run the [Core IT][core-its] successfully.

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [X] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [X] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

[core-its]: https://maven.apache.org/core-its/core-it-suite/

### CODE REVIEW

**Review by michael-o — COMMENTED** [review]
Where did you get the default value from?

**Comment by sungpeo** [comment]
@michael-o 
Sorry, I made a mistake.
Default was "fail".

https://github.com/apache/maven/blob/a1ba33069fad1fb9c6e9cd458ad233ff3a74aadd/maven-artifact/src/main/java/org/apache/maven/artifact/repository/ArtifactRepositoryPolicy.java#L40-L46

File: maven-compat/src/main/mdo/profiles.mdo [hunk 1]
```diff
@@ -308,7 +308,7 @@ under the License.
           <name>checksumPolicy</name>
           <version>1.0.0</version>
           <description>What to do when verification of an artifact checksum fails - warn, fail, etc. Valid values are
-            "fail", "warn" (default) or "ignore"</description>
+            "fail" (default), "warn" or "ignore"</description>
```
Comment (gnodet) [inline maven-compat/src/main/mdo/profiles.mdo:311]: Given we list all the values, should we really keep the `- warn, fail, etc.` part after the `What to do when verification of an artifact checksum fails` sentence ?
Maybe simply:
`What to do when verification of an artifact checksum fails. Valid values are "fail" (default), "warn" or "ignore".`

File: maven-settings/src/main/mdo/settings.mdo [hunk 2]
```diff
@@ -969,7 +969,7 @@
           <version>1.0.0+</version>
           <description>
             What to do when verification of an artifact checksum fails -
-            warn, fail, etc. Valid values are "fail", "warn" (default) or "ignore".
+            warn, fail, etc. Valid values are "fail" (default), "warn" or "ignore".
```
Comment (gnodet) [inline maven-settings/src/main/mdo/settings.mdo:972]: Same here imho.
`What to do when verification of an artifact checksum fails. Valid values are "fail", "warn" (default) or "ignore".`

File: maven-compat/src/main/mdo/profiles.mdo (hunk 1 above)
Comment (michael-o) [reply maven-compat/src/main/mdo/profiles.mdo:311]: Correct, avoid duplication.

Comment (sungpeo) [reply maven-compat/src/main/mdo/profiles.mdo]: @gnodet 
Thanks for the detail correction.

**Review by gnodet — APPROVED** [review]

**Comment by michael-o** [comment]
I need to check whether this should be in the model at all. The default value is set in Java code (Maven), not dictated by the model. The model does not change.

**Comment by michael-o** [comment]
This is how it should look like:
https://github.com/apache/maven/pull/405/files#diff-507b6c79490c5be04e4f368dded08d6134b27b4880e189c62ec43a1dae4930c1

**Comment by sungpeo** [comment]
@michael-o 

You mean it should be like this(below) ?

`What to do when verification of an artifact checksum fails. Valid values are "fail" (default for Maven 4 and above), "warn" (default for Maven 2 and 3) or "ignore".`

**Comment by michael-o** [comment]
Correct.

**Comment by sungpeo** [comment]
@michael-o 
I created a new commit for the default value.

**Pull request closed without merging**